src/BerconCommon.h

```diff
@@ -139,208 +163,239 @@ enum {
 		p_default, 0.f, p_range, -1000000.0f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_OFF_Y, IDC_OFF_Y_SPIN, SPIN_AUTOSCALE,
 		p_end,
-	xyz_offset_z, _M("xyz_offset_z"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_OFFSET_Z,
+		xyz_offset_z, _T("xyz_offset_z"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_OFFSET_Z,
 		p_default, 0.f, p_range, -1000000.0f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_OFF_Z, IDC_OFF_Z_SPIN, SPIN_AUTOSCALE,
 		p_end,
 
-	xyz_size_x, _M("xyz_size_x"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_SIZE_X,
+		xyz_size_x, _T("xyz_size_x"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_SIZE_X,
 		p_default, 1.f, p_range, 0.0001f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_SIZ_X, IDC_SIZ_X_SPIN, SPIN_AUTOSCALE,
 		p_end,
-	xyz_size_y, _M("xyz_size_y"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_SIZE_Y,
+		xyz_size_y, _T("xyz_size_y"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_SIZE_Y,
 		p_default, 1.f, p_range, 0.0001f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_SIZ_Y, IDC_SIZ_Y_SPIN, SPIN_AUTOSCALE,
 		p_end,
-	xyz_size_z, _M("xyz_size_z"), TYPE_FLOAT, P_ANIMATABLE,IDS_XYZ_SIZE_Z,
+		xyz_size_z, _T("xyz_size_z"), TYPE_FLOAT, P_ANIMATABLE,IDS_XYZ_SIZE_Z,
 		p_default, 1.f, p_range, 0.0001f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_SIZ_Z, IDC_SIZ_Z_SPIN, SPIN_AUTOSCALE,
 		p_end,
 
-	xyz_angle_x, _M("xyz_angle_x"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_X,
+		xyz_angle_x, _T("xyz_angle_x"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_X,
 		p_default, 0.f, p_range, -1000000.0f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_ANG_X, IDC_ANG_X_SPIN, SPIN_AUTOSCALE,
 		p_end,
-	xyz_angle_y, _M("xyz_angle_y"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_Y,
+		xyz_angle_y, _T("xyz_angle_y"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_Y,
 		p_default, 0.f, p_range, -1000000.0f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_ANG_Y, IDC_ANG_Y_SPIN, SPIN_AUTOSCALE,
 		p_end,
-	xyz_angle_z, _M("xyz_angle_z"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_Z,
+		xyz_angle_z, _T("xyz_angle_z"), TYPE_FLOAT, P_ANIMATABLE, IDS_XYZ_ANGLE_Z,
 		p_default, 0.f, p_range, -1000000.0f, 1000000.0f,
 		p_ui, TYPE_SPINNER, EDITTYPE_FLOAT, IDC_ANG_Z, IDC_ANG_Z_SPIN, SPIN_AUTOSCALE,
 		p_end,
 
-	xyz_tile_x, _M("xyz_tile_x"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, x, p_ui, TYPE_INTLISTBOX, IDC_TIL_X, 0, p_end,
-	xyz_tile_y, _M("xyz_tile_y"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, y, p_ui, TYPE_INTLISTBOX, IDC_TIL_Y, 0, p_end,
-	xyz_tile_z, _M("xyz_tile_z"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, z, p_ui, TYPE_INTLISTBOX, IDC_TIL_Z, 0, p_end,
+// 	BUG FIX FEB 2024: TILE_Y and TILE_Z were never referenced
+//	xyz_tile_y, _T("xyz_tile_y"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, y, p_ui, TYPE_INTLISTBOX, IDC_TIL_Y, 0, p_end,
+//	xyz_tile_z, _T("xyz_tile_z"), TYPE_INT, P_ANIMATABLE, IDS_XYZ_TILE_X, p_default, z, p_ui, TYPE_INTLISTBOX, IDC_TIL_Z, 0, p_end,
```

Comment:
Could you explain this to me? I haven't read this code base indepth for a long time so bare with me :)

Reply:
I actually just realized it was in two places...
![image](https://github.com/Bercon/BerconMaps/assets/37090381/57c6c13a-adc9-4371-b2c8-906e7b758f27)